products/knownprojects/sql/aggregate/longform_cd_output.sql

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************************************************************************************************************************
-Sources kpdb - finalized version of KPDB build
+Sources kpdb_deduplicated - finalized version of KPDB build with no duplicatre record_id values
```

Comment:
Nitty nit but there are many `duplicatre`s in these comments (similar vibe to socrate actually)

Reply:
and `survery`! my middle finger is too trigger happy

table descriptions in sql files might be non-ideal anyway if they can be described in a yml. will fix